pkg/subctl/cmd/test.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onsi/ginkgo/config"
+	"github.com/submariner-io/shipyard/test/e2e"
+	_ "github.com/submariner-io/submariner-operator/pkg/subctl/benchmark"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	testCmd = &cobra.Command{
+		Use:   "test",
```

Comment:
test->benchmark? I added the suggestion to the enhancement proposal. Testing is more checking if it's ok or not... benchmarking is getting a report of capabilities/performance.

Reply:
yeah, makes sense.